juniper_official/interface/collect-logical-interface-stats.rule

```diff
@@ -0,0 +1,252 @@
+/*
+ * Collect Logical Interface traffic statistics for pathfinder dynamic topology.
+ *
+ * Req 60: https://paragon-automation.atlassian.net/browse/REQ-60
+ *
+ * Two inputs control detection
+ *
+ *   1) input-ifd-name, is a regular expression that matches the interfaces
+ *      that you would like to monitor.  By default it's '.*', which matches
+ *      all interfaces. Use something like 'ge.*' to match only gigabit
+ *      ethernet interfaces. 
+ *
+ *   2) input-ifl-no, is a regular expression that matches the IFL index number that
+ *      you would like to monitor.  By default it's '.*', which matches all
+ *      interfaces. Use something like '0-10*' to match only 0 to 10 IFLs.
+ *
+ */
+ healthbot {
+    topic interfaces {
+		rule collect-logical-interface-stats {
+            keys [ ifl-name interface-name sub-interface-index ];
```

Comment:
Making non sensor field(ifl-name) is not a tested feature in EOP. I would suggest to remove this and handle this in business service.